runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java

```diff
@@ -54,19 +56,21 @@
 @PluginDescriptor(
 	name = "Barbarian Assault",
 	description = "Show a timer to the next call change and game/wave duration in chat.",
-	tags = {"minigame", "overlay", "timer"}
+	tags = {"minigame", "overlay", "timer", "notifications"}
 )
 public class BarbarianAssaultPlugin extends Plugin
 {
 	private static final int BA_WAVE_NUM_INDEX = 2;
 	private static final String START_WAVE = "1";
 	private static final String ENDGAME_REWARD_NEEDLE_TEXT = "<br>5";
+	static final int CALL_DURATION = 30;
 
 	private Font font;
 	private Image clockImage;
 	private int inGameBit = 0;
 	private String currentWave = START_WAVE;
 	private GameTimer gameTime;
+	private boolean playingCue = false;
```

Comment:
`false` is the default value for boolean fields, so it can be omitted.
```suggestion
	private boolean playingCue;
```

Reply:
Also, this value should be reset to `false` if either the plugin or `config.notifyCallChange()` are disabled.